src/Umbraco.Web.UI.Client/src/views/propertyeditors/email/email.html

```diff
@@ -1,6 +1,5 @@
 ﻿<div>
-
-    <input type="text" name="textbox"
+    <input type="email" name="textbox"

```

Comment:
I have changed to a ```type="email"```as it will give a different (optimized) keypad on mobile and tablet.